.github/workflows/release.yml

```diff
@@ -2,6 +2,11 @@ name: Release
 
 on:
   workflow_dispatch:
+    inputs:
+      target_branch:
+        description: "Branch to release from. This is usually for backporting fixes to earlier versions."
+        required: true
+        default: "main"
```

Comment:
Do we need this? `workflow_dispatch` already enables triggering from any branch. I guess if would want to checkout from a specific tag but we are planning on having the backport branch pushed back to origin no? 🤔 

<img width="396" alt="image" src="https://github.com/user-attachments/assets/d96f19eb-c242-4b20-b5e6-19b025a8416d" />

Reply:
You're right! I didn't notice that branch was already a choice. I'll revert the changes to this workflow.